### ISSUE TITLE
URGENT: Prevent false-negative FAILs on VFs

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#2214]
## Observed behaviour

https://github.com/google/fonts/pull/1743#issuecomment-440762696 has FAILs for things which are OK

## Expected behaviour

FontBakery should very early detect if the font is a VF or not, and if so, then pass that as some kind of flag into the checks, and then checks should work appropriately if the font is a VF, such as allowing PS names, etc

**Comment by felipesanches** [comment]
yes, we do have a condition called `is_variable_font` that can be used for that purpose.

**Comment by davelab6** [comment]
https://github.com/thundernixon/Encode-Sans/issues/1#issuecomment-440830836 also shows some things which are false negatives

**Comment by felipesanches** [comment]
> thundernixon/Encode-Sans#1 (comment) also shows some things which are false negatives

Can you clarify which things? I only saw the var font naming false negatives there. What else is there?

**Comment by davelab6** [comment]
com.google.fonts/check/043 is described in another issue

**Comment by felipesanches** [comment]
Which other issue? I search for it and could not find it.

**Comment by thundernixon** [comment]
Probably, it sounds like we should update the checks to accept `2000` as an acceptable UPM for variable fonts.

> 🔥 FAIL: Checking unitsPerEm value is reasonable.
> ⚠️ WARN: Is font em size (ideally) equal to 1000?

**Comment by felipesanches** [comment]
alright! Got it! It is #2185
